server/routes/conversationRoute.js

```diff
@@ -0,0 +1,57 @@
+const express = require("express");
+const router = new express.Router();
+
+const User = require("../models/User");
+const Conversation = require("../models/Conversation");
+const verifyToken = require("../middlewares/verifyToken");
+
+// Create a conversation
+router.post("/conversations/new/", verifyToken, async (req, res) => {
+	try {
+		// Retrieving the ._ids of the person initiating and receiving the conversation
+		const sender = await User.findById({ _id: req.user._id });
```

Comment:
looks like you only ever use this data for its id. So you can save this DB call and just use `req.user._id` since it's already verified